russian/buildings-dep/buildings-dep.md

```diff
@@ -1,4 +1,8 @@
 Вы дозвонились в ЖЭК, хотя и боялись, что вас не поймут. Объяснили
 ситуацию и попросили помочь. Вас внимательно выслушали, нос казали
 что все сейчас заняты, так как Ваш звонок такой не первый за день и
-продеться подождать.
\ No newline at end of file
+продеться подождать. Что вы будете делать?
+
+[Вы чуете неладное и решаете включить телевизор.](switch-on-tv/switch-on-tv.md)
+
+[Вас одолевает паника, вы хватаете телефон и звоните маме.](call-mum.md/call-mum.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.